x-pack/heartbeat/monitors/browser/suite_test.go

```diff
@@ -114,3 +115,37 @@ func TestEmptySource(t *testing.T) {
 	require.Regexp(t, ErrBadConfig(source.ErrInvalidSource), e)
 	require.Nil(t, s)
 }
+
+func TestExtraArgs(t *testing.T) {
+	tests := []struct {
+		name string
+		cfg  *Config
+		want []string
+	}{
+		{
+			"no args",
+			&Config{},
+			nil,
+		},
+		{
+			"sandbox",
+			&Config{Sandbox: true},
+			[]string{"--sandbox"},
+		},
+		{
+			"kitchen sink",
+			&Config{SyntheticsArgs: []string{"--capability", "trace"}, Sandbox: true},
```

Comment:
I am curios to see a test with capability > 2 and other flags as part of synthetic args. Reason being variadic args are prone to errors.